content/modules/flow-ui/pages/vc/layout-rules.adoc

```diff
@@ -677,3 +677,66 @@ An example of defining a responsive text field that has fixed size on big screen
 
 image::layout-rules/responsive-text-field.gif[]
 
+
+An example of defining a form layout with a single column that has fixed size on big screens and 100% width on small ones:
+
+[source,xml]
+----
+<formLayout id="form"
+            dataContainer="taskTypeDc"
+            classNames="mx-m"
+            maxWidth="40em"> <!--2-->
+    <responsiveSteps>
+        <responsiveStep minWidth="0" columns="1"/> <!--1-->
+    </responsiveSteps>
+
+    <textField id="nameField" property="name"/>
+    <textArea id="descriptionField" property="description"
+              height="9.5em"/>
+</formLayout>
+----
+<1> Defining that the form layout must have a single column for all layout sizes
+<2> Setting the maximum width
+
+image::layout-rules/single-column-formLayout.gif[]
+
+For simple styling *Lumo Utility Classes* can be used. *Lumo Utility Classes* are predefined CSS class names and stylesheets. They can be used to style HTML elements and layouts without having to write CSS.
+
+Each utility class applies a particular style to the element, such as background color, borders, fonts, sizing, or spacing. Classes for applying CSS flexbox and grid layout features are also available.
+
+image::layout-rules/LumoUtility.png[]
+
+An example of using lumo utility classes to add a background color to a container:
+
+[source,xml]
+----
+<vbox id="imageWrapper"
+      classNames="border                <!--1-->
+                  rounded-m             <!--2-->
+                  border-contrast-20"   <!--3-->
+      alignItems="CENTER"
+      width="100%" maxWidth="30em">
+----
+<1> Adds border with default border color
+<2> Sets border radius equal to `var(--lumo-border-radius-m)`
+<3> Sets border color equal to `var(--lumo-contrast-20pct)`
+
+image::layout-rules/LumoUtility-example.png[]
```

Comment:
What about a link to Vaadin sources instead of the image?
Or just an excerpt from them?